presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchClient.java

```diff
@@ -101,9 +103,39 @@ public ElasticsearchClient(ElasticsearchTableDescriptionProvider descriptions, E
 
         for (ElasticsearchTableDescription tableDescription : tableDescriptions.getAllTableDescriptions()) {
             if (!clients.containsKey(tableDescription.getClusterName())) {
-                Settings settings = Settings.builder().put("cluster.name", tableDescription.getClusterName()).build();
+                Settings settings;
                 TransportAddress address = new TransportAddress(InetAddress.getByName(tableDescription.getHost()), tableDescription.getPort());
-                TransportClient client = new PreBuiltTransportClient(settings).addTransportAddress(address);
+                TransportClient client;
+                switch (config.getCertificateFormat().toUpperCase()) {
+                    case "PEM":
+                        settings = Settings.builder()
+                                .put("client.transport.ignore_cluster_name", true)
+                                .put(SSLConfigConstants.SEARCHGUARD_SSL_TRANSPORT_PEMCERT_FILEPATH, config.getPemcertFilepath())
```

Comment:
static import SEARCHGUARD_SSL_TRANSPIRT_PERMCERT_FILEPATH
the same for all constants below